src/vma/util/verbs_extra.h

```diff
@@ -308,7 +317,7 @@ static inline void init_vma_ibv_cq_init_attr(vma_ibv_cq_init_attr* attr)
 {
 #ifdef DEFINED_IBV_EXP_CQ_TIMESTAMP
 		attr->flags = IBV_EXP_CQ_TIMESTAMP;
-		attr->comp_mask = IBV_EXP_CQ_INIT_ATTR_FLAGS;
+		attr->comp_mask |= IBV_EXP_CQ_INIT_ATTR_FLAGS;
```

Comment:
This is good. Please do it for `attr->flags` (above) as well.

#### Introduced in:
MP_RQ: add cq_mgr_mp and qp_mgr_mp